scripts/ops/README.md

```diff
@@ -0,0 +1,40 @@
+# Armadillo for OPS
+
+To monitor and control Armadillo and its DataShield docker containers we need to use the Armadillo API.
+
+## Armadillo Controller
+
+With `armadilloctl` you can control the DataShield docker containers.
```

Comment:
who is the envisioned user for this? I mean why is this outside of the app?

Reply:
From hindsight a checkbox in Armadillo was probably enough ... the code was already written as investigation/spinoff from the Rock only PR.